AmplifyPlugins/DataStore/DataStoreCategoryPlugin.xcodeproj/xcshareddata/xcschemes/AWSDataStoreIntegrationLocalTests.xcscheme

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1320"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "YES">
+   </BuildAction>
+   <TestAction
+      buildConfiguration = "Debug"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
+      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      shouldUseLaunchSchemeArgsEnv = "YES">
+      <Testables>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "2149E61C23886CEE00873955"
+               BuildableName = "AWSDataStoreCategoryPluginIntegrationTests.xctest"
+               BlueprintName = "AWSDataStoreCategoryPluginIntegrationTests"
+               ReferencedContainer = "container:DataStoreCategoryPlugin.xcodeproj">
+            </BuildableReference>
+            <SkippedTests>
```

Comment:
Are these intentional? I see skipped tests in all xcsheme

Reply:
yup, most tests in AWSDataStoreCategoryPluginIntegrationTests are skipped, and split into individual schemas that enable one or more test classes from the original target

Reply:
nit: if they are used in other scheme, should we then delete it from the scheme where it is skipped?

Reply:
went over to try this, and don't think we can in the current structure
![image](https://user-images.githubusercontent.com/1365977/165383939-8ef27dfc-faa0-4991-860f-d6440ca9ed86.png)

Each scheme is under a particular target, which inherits all of the test classes of that target. Having multiple targets is another option, and then 1-1 schema to target would clean it up. I believe this should be what we should aim for, but each scheme only contains a few tests, it's much more maintainable at the moment to keep them all under a single target. Once we improve on each schema's test and add more coverage, then we can migrate them to its own separate target (like we do with the other two targets MultiAuth and Flutter)